quickcomm/apiurls.py

```diff
@@ -11,10 +11,34 @@
 author = AuthorRouter()
 author.register(r'authors', api.AuthorViewSet)
```

Comment:
do we want to be updating author and post to look like how all the other objects look?